spec/controllers/shared_examples/shared_spec_for_sync_controller.rb

```diff
@@ -211,6 +219,88 @@ def set_authentication_headers
   end
 end
 
+RSpec.shared_examples 'a working Current sync controller sending records' do
```

Comment:
Should we make separate files for V1 and current controller?

Reply:
Will extract when we have more shared specs for current. Right now, these specs have more things in common than not.